### PR TITLE
Create Setting-CPI-Requirements-CBRS-Installations.md

### PR DESCRIPTION
**Pull request by mrfizzy99**
This HIP aims to set CPI (Certified Professional Installer) requirements for CBRS deployments to better the installation quality and knowledge of the deployers for the CBRS network.  Every new cbrs deployment and/or re-registration of a cbrs radios, and will require a CPI identification number when submitting for SAS registration.

### CODE REVIEW

File: Setting-CPI-Requirements-CBRS-Installations.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Setting-CPI-Requirements-CBRS-Installations
```
Comment (abhay) [inline Setting-CPI-Requirements-CBRS-Installations.md:1]:
```suggestion
# HIP XXX: Setting CPI Requirements for CBRS Installations
```

File: Setting-CPI-Requirements-CBRS-Installations.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- Author(s): @mrfizzy99
+- Start Date: 2024/02/07
+- Category: Technical
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
+- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
+- Vote Requirements: Current veMOBILE Holders
```
Comment (abhay) [inline Setting-CPI-Requirements-CBRS-Installations.md:8]:
```suggestion
- Author(s): [@mrfizzy99](https://github.com/mrfizzy99)
- Start Date: 2024-02-07
- Category: Economic, Technical
- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
- Vote Requirements: veMOBILE Holders
```

**Review by abhay — COMMENTED** [review]
Let's get this ready for a merge so discussion can start. Just to be clear, are you stating that _all_ new CBRS deployers must be a CPI? I think it'd help the HIP if that was more clear. Also is there any requirement that Nova uses the CPI ID directly or do they still get to use theirs for the SAS? It would also be helpful if the network knew who was ultimately responsible for the SAS records.

**Comment by mrfizzy99** [comment]
@abhay 
"Just to be clear, are you stating that all new CBRS deployers must be a CPI?"
Any new SAS that needs to be done , regardless of the reason, will be require a CPI public key.  

"Also is there any requirement that Nova uses the CPI ID directly or do they still get to use theirs for the SAS? "
Their is no requirement for the Service Provider to actually use this in the SAS registration in the backend at this time. They can choose to use it in SAS, or just take note of it for future reference. 

The point of this HIP is solely to attempt better quality, experience, and education of the people that decide on CBRS deployment. 


Overall, I have been told that the Service Provider can require this regardless of a HIP, and no HIP is needed.   
I want to assume it was never wanted or required to ease the deployment success of CBRS, as their are less barriers to entry. But we are at the next point of deployment in the ecosystem which is wifi, and wifi can be deployed at a greater benefit and ease than CBRS should be.

**Comment by abhay** [comment]
I like the specificity. Add those details to the HIP and let's get it merged for discussion?

**Comment by mrfizzy99** [comment]
Will do.  

Pull 910 is way more important atm.  Please review that one for expediting.

**Comment by waveform06** [comment]
@mrfizzy99 Whats the status with this now? Waiting on the results of other HIPs/PRs or do you think this should proceed.

**Comment by mrfizzy99** [comment]
> @mrfizzy99 Whats the status with this now? Waiting on the results of other HIPs/PRs or do you think this should proceed.

Waiting for @heatedlime to suggest updates as he showed interest in modifying. 👴

**Comment by heatedlime** [comment]
> > @mrfizzy99 Whats the status with this now? Waiting on the results of other HIPs/PRs or do you think this should proceed.
> 
> Waiting for @heatedlime to suggest updates as he showed interest in modifying. 👴

Before I do anything, I’d like to see if 113 passes or not. Don’t let me hold anything up.

**Comment by heatedlime** [comment]
> @mrfizzy99 @heatedlime Any updates going to happen on this one?

None from me; I’m no longer interested in it.

**Comment by waveform06** [comment]
This HIP then really needs to describe how the new process works within Nova - which will have to come from Nova Labs.
If they don't want to implement it, it needs a new registrar then.

**Comment by DigidyDOG** [comment]
Can I throw out a question since its not clear in the text, are both courses required? Can we make it clear that if this hip does head towards implementation, that both courses are not free and do cost hundreds of dollars? Don't get me wrong I think its an interesting idea and i'd be glad to take it to learn more about the antennas. Idea; Bonus rewards to hotspot owners that are CPI certified and add it as another oracle.

**Comment by waveform06** [comment]
@DigidyDOG As I understand it its not about the number of courses. Its about the CPI ID you get after passing the exam.
Both links seem to end up at the same place. Or you may not even need a course. Just take the exam for the cert
The standards for operating a CPI training course were updated last week about recording certs
https://winnf.memberclicks.net/assets/CBRS/WINNF-TS-0247.pdf

**Comment by waveform06** [comment]
There seems to be places offering the course+exam for under $500

**Review by devilish66694 — APPROVED** [review]

**Review by devilish66694 — APPROVED** [review]

**Comment by waveform06** [comment]
> This HIP then really needs to describe how the new process works within Nova - which will have to come from Nova Labs.
> If they don't want to implement it, it needs a new registrar then.

@mrfizzy99 Have you talked to Nova about engaging on this, looked for a new registrar or should this PR get closed

**Comment by abhay** [comment]
Marking this for archive at the end of the week.

**Comment by abhay** [comment]
ready for archive. can be reopened if necesary.

**Pull request closed without merging**